pkg/agent/sidecar/handler/grpc/option.go

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package grpc provides grpc server logic
+package grpc
+
+import (
+	"github.com/vdaas/vald/pkg/agent/sidecar/service"
+)
+
+type Option func(*server)
+
+var (
+	defaultOpts = []Option{}
+)
+
+func WithStorageObserver(so service.StorageObserver) Option {
+	return func(s *server) {
+		if i != nil {
```

Comment:
The build is failing caused by this `i`. Please fix it :wink:
https://github.com/vdaas/vald/pull/404/checks?check_run_id=706185779